src/libs/actions/Policy.ts

```diff
@@ -3919,7 +3919,9 @@ function navigateWhenEnableFeature(policyID: string, featureRoute: Route) {
     new Promise<void>((resolve) => {
         resolve();
     }).then(() => {
-        Navigation.navigate(featureRoute);
+        setTimeout(() => {
```

Comment:
Do you have any other solution for this? We avoid using setTimeout

Reply:
Let's take a look at my video, I posted. I waited for all the request were called, so why are you saying that 
> Our request to enable this feature will not be completed because it was in the queue

Reply:
I don't think the your explanation [here](https://github.com/Expensify/App/pull/40219/files#r1566881269) is correct. Let's discuss our situation here, all APIs were called and we still saw the NotFound screen. That shouldn't happen and I think we can avoid using timeout to fix it.

Reply:
Any update here?

Reply:
Yeah yeah, np. Please keep me posted